src/main.ts

```diff
@@ -66,6 +67,12 @@ export class DreamsSDK {
 
     if (localeInput) localeInput.setAttribute('value', locale);
 
+    const locationInput: HTMLInputElement = (this.form.querySelector(
```

Comment:
codeclimate complains about double quotes. I don't agree with that but I guess we should stick to what's been set up for other projects

Reply:
Where is codeclimate running?

Reply:
do you have access to it? like to this link for example: https://codeclimate.com/repos/61487d133aebbd01a000b141

Reply:
I unfortunately don't have access to that :(

Reply:
You should be able to login with your github account?

Reply:
You should be able to login with your github account?

Reply:
I get access denied from codeclimate. But I can run it locally, which is not optimal, but it works. I have a memory codeclimate has a max limit on users on some plans, no?